### PR TITLE
Added content about using Teku with Eth2Signer.

### PR DESCRIPTION
**Pull request by bgravenorst**
Signed-off-by: Byron Gravenorst <byron.gravenorst@consensys.net>

## Pull Request Description
Added content about using Teku with Eth2Signer.

### CODE REVIEW

File: docs/HowTo/Get-Started/Use-External-Signer.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+The command line:
+
+* Specifies the validator public keys for which Eth2Signer signs attestations and blocks using
+    [`--validators-external-signer-public-keys`](../../Reference/CLI/CLI-Syntax.md#validators-external-signer-public-keys).
+* Specifies the url of the Eth2Signer client using
+    [`--validators-external-signer-url`](../../Reference/CLI/CLI-Syntax.md#validators-external-signer-url).
+
+!!! note
+    Create a signing key configuration file for each public key specified using
```
Comment (jframe) [inline docs/HowTo/Get-Started/Use-External-Signer.md:33]: Not sure what is meant by this. Are you referring to the configuration file in Eth2Signer that is needed for each key?

File: docs/HowTo/Get-Started/Use-External-Signer.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+* Eth2Signer installed and running
+* Configured signing key configuration files
+
+## Start Teku
+
+Start Teku and specify the external signer options.
+
+```bash
+teku --eth1-deposit-contract-address=dddddddddddddddddddddddddddddddddddddddd \
+--eth1-endpoint=http://localhost:8545 --validators-key-file=validator_keys \
```
Comment (jframe) [inline docs/HowTo/Get-Started/Use-External-Signer.md:20]: I think using both the validators keys from the key file and using an external signer together is an uncommon scenario so would probably remove the --validators-key-file=validator_keys from the example

File: docs/HowTo/Get-Started/Use-External-Signer.md (hunk 1 above)
Comment (bgravenorst) [reply docs/HowTo/Get-Started/Use-External-Signer.md:33]: Yes I am. Will try and rephrase this to make more clear.

File: docs/HowTo/Get-Started/Use-External-Signer.md (hunk 2 above)
Comment (bgravenorst) [reply docs/HowTo/Get-Started/Use-External-Signer.md:20]: Done

File: docs/HowTo/Get-Started/Use-External-Signer.md (hunk 1 above)
Comment (bgravenorst) [reply docs/HowTo/Get-Started/Use-External-Signer.md:33]: Will link to the appropriate Eth2Signer content when that content is merged.

File: docs/HowTo/Get-Started/Use-External-Signer.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+---
+
+# Use an external signer
+
+Teku supports the [Eth2Signer] external signing client.
+
+**Prerequisites**:
+
+* Eth2Signer installed and running
+* [Signing key configuration files].
```
Comment (MadelineMurray) [inline docs/HowTo/Get-Started/Use-External-Signer.md:12]: Should there be fullstops on both or neither of these?

File: docs/HowTo/Get-Started/Use-External-Signer.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+The command line:
+
+* Specifies the validator public keys for which Eth2Signer signs attestations and blocks using
+    [`--validators-external-signer-public-keys`](../../Reference/CLI/CLI-Syntax.md#validators-external-signer-public-keys).
+* Specifies the url of the Eth2Signer client using
+    [`--validators-external-signer-url`](../../Reference/CLI/CLI-Syntax.md#validators-external-signer-url).
+
+!!! note
+    Create a signing key configuration file for each public key specified using
```
Comment (MadelineMurray) [inline docs/HowTo/Get-Started/Use-External-Signer.md:33]: Should this link somewhere describing how to create a signing key config file?
